test/distributed/test_store.py

```diff
@@ -157,10 +157,7 @@ def _create_store(self):
         return store
 
     def test_address_already_in_use(self):
-        if sys.platform == "win32":
-            err_msg_reg = "Only one usage of each socket address*"
-        else:
-            err_msg_reg = "^Address already in use$"
+        err_msg_reg = "^The server socket has failed to listen on any local "
```

Comment:
Are we going to print more precise error messages else where regarding what the failure is? Besides, does this mean we expect a different error here than "Address already in use"?

Reply:
The regex just tests the start of the error string. A full error string that the user will see, would look like:
```
The server socket has failed to listen on any local network address. The server socket has failed to bind to [::]:29501 (generic error: 98 - Address already in use). The server socket has failed to bind to 0.0.0.0:29501 (generic error: 98 - Address already in use).
```